Update configuration.mdx (#33420)
hashicorp/terraform: website/docs/language/settings/backends/configuration.mdx

- **File**: A configuration file may be specified via the `init` command line.
  To specify a file, use the `-backend-config=PATH` option when running
  `terraform init`. You need an existing backend configuration that you can override with
  the config file you provide in PATH. The config file must contain only the key/value pairs
  for your backend. For example, run `terraform init -backend-config="./state.config"` to 
  override your backend configuration defined in `state.tf` with the values from `state.config`: 

  ```hcl
  # state.tf
  terraform {
    backend "s3" {
      bucket = "" 
      key    = ""
      region = ""
      profile= ""
    }
  }
  ```

  ```hcl
  # state.config
  bucket = "your-bucket" 
  key    = "your-state.tfstate"
  region = "eu-central-1"
  profile= "Your_Profile"
  ```

>   Important: If your config file contains secrets it may be kept in

Reviewer: ```suggestion
  When your configuration file contains secrets, you can store them in
```
Presumably, this is all important or else we wouldn't have written it down :)
